README.md

```diff
@@ -43,6 +43,36 @@ python nutrition_labels/grant_tagger.py --training_data_file data/processed/trai
 
 where `vectorizer_type` can be 'count', 'tfidf' or 'bert', `model_type` can be 'naive_bayes', 'SVM' and 'log_reg', and `bert_type` (if using bert) can be 'bert' and 'scibert'.
 
+## finding_most_common_cohort.py / finding_most_common_references.py 
+
+To find the most common health data sets first run:
+
+```
+python nutrition_labels/finding_most_common_cohort.py
+```
+
+This uses data extracted from EPMC which has all the citations from all the papers produced from tech grants. 
```

Comment:
I'd change this to:

This uses data extracted from EPMC ('data/processed/tech_grants_references_dict_2020-09-16.json' and 'data/processed/tech_grants_reference_ids_counter_2020-09-16.json') which has all the **references** from all the papers produced from tech grants.

just to avoid confusion between the words citations and references.

I'll add a PR to add my code of where I make these and also add to the readme later!